lib/utils/analytics/generatePayload.js

```diff
@@ -85,5 +93,11 @@ module.exports = async (serverless) => {
     npmDependencies,
     triggeredDeprecations: Array.from(triggeredDeprecations),
     versions,
+    createdAt: Date.now(),
```

Comment:
`createdAt` feels as something describing mutable data (e.g. on files we have `createdAt`, `modifiedAt` etc.), whille here we're signalling where this event happened. Wouldn't it be more accurate to simply name it `timestamp` or `time` ?

Reply:
you're totally right - timestamp feels much better here, I'll change it